src/data-workspace/inputs/option-set.js

```diff
@@ -21,21 +27,43 @@ export const OptionSet = ({
     onFocus,
     disabled,
     locked,
+    multi,
 }) => {
+    const parse = (value) => {
+        console.log('parse', value)
+        if (multi) {
+            return (value && value.join(',')) || ''
+        } else {
+            // Empty values need an empty string
+            return value || ''
+        }
+    }
+
     const {
         input,
         meta: { data },
     } = useField(fieldname, {
         subscription: { value: true, data: true },
+        format:
+            // format to an array when multi, since component needs an array
+            multi
```

Comment:
this could be a bit more concise, e.g.
`multi ? (value) => ((value && value.split(',')) || []) : undefined`
or similar, but obviously just a stylistic choice up to you

Reply:
Yup, originally did it like that, but changed to explicit return for the log statement while debugging.

Reply:
Will keep it like this, since we now sort the result of format. And I think something like 
```js
((value && value.split(MULTI_TEXT_SEPERATOR)) || []).sort(sortByOptionsOrder)
```

is harder to read.